README.md

```diff
@@ -141,6 +143,10 @@ making a new tag during step 2. This is essentially redeploying an older version
 
    1. update the `production` branch: `git checkout production && git merge master && git push origin production --tags`
 
+1. send a message to the `#buttons-aa-general` Slack channel letting everyone know that you are doing a deployment and to expect some downtime.
+
+1. send a text message to all of the Responder phones letting them know that you are doing some routing maintenance, there will be downtime, and that you will notify them when everything is back to normal again. For example: "Notice: Your Brave Buttons System is down for maintenance. During this time, Button presses will not trigger an alert. You will receive another text message when everything is back online. Thank you for your patience. Have a nice day."
```

Comment:
typo here: 'routing' should be 'routine'

Reply:
Fixed! Thanks.